web/src/timerender.ts

```diff
@@ -355,6 +389,21 @@ export function get_markdown_time_tooltip(reference: HTMLElement): DocumentFragm
     return "";
 }
 
+function update_recent_topics_timestamps(): void {
```

Comment:
Should this check `recent_topics_util.is_visible()` before doing anything? I think we can likely discard the update list if it isn't.

Reply:
I attempted to use this util function, but I encountered the error `Cannot resolve file 'recent_topics_util'`. There may be an issue with importing .js files from .ts files, or it could be related to a dependency cycle. For now, I have written an if statement to check if the element is visible.

Reply:
I've added a comment explaining the reason for not using `recent_topics_util.is_visible()` in this case.